Resources/doc/index.md

```diff
@@ -52,8 +52,8 @@ lexik_jwt_authentication:
 Configure your `parameters.yml.dist` :
 
 ``` yaml
-    jwt_private_key_path: %kernel.root_dir%/var/jwt/private.pem   # ssh private key path
-    jwt_public_key_path:  %kernel.root_dir%/var/jwt/public.pem    # ssh public key path
+    jwt_private_key_path: %kernel.root_dir%/../var/jwt/private.pem   # ssh private key path
+    jwt_public_key_path:  %kernel.root_dir%/../var/jwt/public.pem    # ssh public key path
     jwt_key_pass_phrase:  ''                                      # ssh key pass phrase
```

Comment:
Sure, this is not what i meant.
The ending comment (i.e. `# ssh key pass phrase`) should be __aligned__ to the previous ones (i.e. `# ssh ...`). e.g:

```diff
- jwt_private_key_path: %kernel.root_dir%/../var/jwt/private.pem   # ssh private key path
- jwt_public_key_path:  %kernel.root_dir%/../var/jwt/public.pem    # ssh public key path
- jwt_key_pass_phrase:  ''                                      # ssh key pass phrase
+ jwt_private_key_path: %kernel.root_dir%/../var/jwt/private.pem   # ssh private key path
+ jwt_public_key_path:  %kernel.root_dir%/../var/jwt/public.pem    # ssh public key path
+ jwt_key_pass_phrase:  ''                                         # ssh key pass phrase
```